kod/object/item.kod

```diff
@@ -613,6 +613,30 @@ messages:
       return 1;
    }
 
+   GetItemRarityGrade(what = $)
+   "Returns the appropriate text coloring tier for the item."
+   {
+   % first check if it's revealed, if not send unrevealed color.
```

Comment:
This is all commented out?

Reply:
Sorry was still working on this, resolving this comment as I now have some code attempts for this function.